File: tests/fixtures/code_samples/ts/sample_038.ts
/** Sample 38: small utility. */

export function operation38(xs: number[]): number {
  let total = 38;
  for (const x of xs) total += x;
  return total;
}

export function operationPure38(value: number): number {
  return (value * 38) % 7919;
}

File: tests/fixtures/code_samples/rust/sample_038.rs
// Sample 38: small utility.
pub fn operation_38(xs: &[i32]) -> i32 {
    let mut total: i32 = 38;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_38(v: i32) -> i32 {
    (v * 38) %% 7919
}

